### DISCUSSION TITLE
ko/docs/integration/java/firezone/

### DISCUSSION THREAD

**Discussion by giscus** [casdoor/casdoor#2483] [Docs comments]
# ko/docs/integration/java/firezone/

Using the OIDC protocol as the IDP to connect various applications, such as FireZone

https://casdoor.org/ko/docs/integration/java/firezone/

**Comment by orlovmyk** [comment]
I keep getting issue described here: https://github.com/firezone/firezone/issues/2607#issuecomment-1801613004
Do you have any suggestions?

**Reply by hsluoyz** [reply]
@orlovmyk "JWT-Empty" only provides the name of the user, what other fields do you need?

![image](https://github.com/casdoor/casdoor/assets/3787410/00b90d9b-d580-47c0-9fd6-f86861d4e26f)

**Reply by orlovmyk** [reply]
@hsluoyz I guess Firezone also needs email at least. By design JWT-empty must provide all non-null fields isn't?

Also using JWT instead of JWT-empty results in such error https://github.com/firezone/firezone/issues/1621

Thanks for reaching me out 🚀

**Reply by hsluoyz** [reply]
@orlovmyk fixed by: https://github.com/casdoor/casdoor/commit/1fb214ae01be870bd86342053754f84fb35e89ff

**Reply by orlovmyk** [reply]
@hsluoyz If it's possible, can you publish casbin/casdoor:v1.451.0 to DockerHub?
I wanna give it a try till setup is available + answer you with resul

**Reply by hsluoyz** [reply]
@orlovmyk try [v1.451.0](https://hub.docker.com/layers/casbin/casdoor/v1.451.0/images/sha256-4487f9b341baf1ac9a1e17a3df6fd63895b7284c2f4db760f723997c3e024b32?context=repo)

**Reply by orlovmyk** [reply]
@hsluoyz yep, issue is fixed
I've managed to log in without any issues

<img width="1724" alt="image" src="https://github.com/casbin/casdoor/assets/29892471/696f4222-13af-42c9-b166-efa451c047ba">

user auto creation is working too
<img width="1356" alt="image" src="https://github.com/casbin/casdoor/assets/29892471/112017de-b406-40d9-99e2-c9d30df1984c">
